expfmt/encode.go

```diff
@@ -37,8 +37,9 @@ func (e encoder) Encode(v *dto.MetricFamily) error {
 }
 
 // Negotiate returns the Content-Type based on the given Accept header.  If no
-// appropriate accepted type is found, FmtText is returned.  Formats can be
-// optionally blacklisted by providing them as blacklisted parameters.
+// appropriate accepted type is found, FmtText is returned (which is the old
```

Comment:
it's the current Prometheus text format, the older Prometheus text format is something else (was that before histograms?)

Reply:
Removed the "old".